src/lua/fiber.c

```diff
@@ -50,6 +50,13 @@ luaL_testcancel(struct lua_State *L)
 	}
 }
 
+void
+fiber_worker_shutdown(void)
+{
+	const char *c = "require('fiber')._internal.worker_shutdown()";
+	VERIFY(luaT_dostring(tarantool_L, c) == 0);
+}
```

Comment:
Can we cancel the fiber from `box.ctl.on_shutdown` trigger?

Reply:
I guess not. At this moment we not close Lua state yet, so we need this fiber to do yielding gc.

Reply:
Discussed f2f and decided to finish this fiber as manageble (see the previous conversation).